lang/en.js

```diff
@@ -166,4 +166,9 @@ export default {
   Checkout: "Checkout",
   ShopAll: "Shop All",
   Advertisment: "Advertisment",
+  "Show Per Page": "Show Per Page",
+  20: "20",
+  40: "40",
+  60: "60",
```

Comment:
remove 20, 40, 60 since we moved it to the config

Reply:
Done and fixed the pagination startIndex issue as well.

Reply:
Resolved